<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>730</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">Configuration Dialog</string>
  </property>
  <widget class="QGroupBox" name="mGroupInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Input</string>
   </property>
   <widget class="QLineEdit" name="mEditInputDir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="mBtnChooseInput">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>46</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Index</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mEditInputIndex">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-1</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="mGroupOutput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>381</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QLineEdit" name="mEditOutputDir">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="mBtnChooseOutput">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mChkAutoOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto output</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="mCmbOutFormat">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>74</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>tiff</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>hdf5</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="mGroupUnpack">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>381</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Unpack</string>
   </property>
   <widget class="QLabel" name="lb1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Board Names</string>
    </property>
   </widget>
   <widget class="QListWidget" name="mLstBoardNames">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Board2004</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Board9054</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lb2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Data Block Size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mEditDataBlockSize">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1300</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pack Size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mEditPackSize">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>8192</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Num DAQ Chns</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mEditNumDaqChnsBoard">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>32</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Num Elements</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mEditNumElements">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>512</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Num Segments</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mEditNumSegments">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>150</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bad Channels</string>
    </property>
   </widget>
   <widget class="QListWidget" name="mLstBadChannels">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>21</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>22</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>23</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>24</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>85</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>86</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>87</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>88</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lb4_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Total firings</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mEditTotFirings">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>180</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="mGroupDisplay">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Preprocess</string>
   </property>
   <widget class="QCheckBox" name="mChkWi">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Wiener</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mChkExact">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Exact</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="mGroupRecon">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>461</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Reconstruction</string>
   </property>
   <widget class="QLabel" name="lb10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed of sound</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="mSpnVm">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>1.510000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lb11">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>km/s</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Initial angle</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="mSpnInitAngle">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="value">
     <number>225</number>
    </property>
   </widget>
   <widget class="QLabel" name="lb13">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>degree</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x size</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="mSpnXSize">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>25</number>
    </property>
   </widget>
   <widget class="QLabel" name="lb15">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y size</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb17">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="mSpnYSize">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>25</number>
    </property>
   </widget>
   <widget class="QLabel" name="lb18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>z size</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb19">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="mSpnZSize">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>9</number>
    </property>
   </widget>
   <widget class="QLabel" name="lb20">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>x center</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb21">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>109</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="mSpnXCenter">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>109</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="lb22">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>y center</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="mSpnYCenter">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>139</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="lb23">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>139</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb24">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sampling freq.</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="mSpnFs">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>40</number>
    </property>
   </widget>
   <widget class="QLabel" name="lb25">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MHz</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb26">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ring radius</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="mSpnR">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>25</number>
    </property>
   </widget>
   <widget class="QLabel" name="lb27">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>50</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb30">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Resolution factor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mEditRf">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb31">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Focal length</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="mSpnLenR">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>80</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>0.000000000000000</double>
    </property>
    <property name="maximum">
     <double>50.000000000000000</double>
    </property>
    <property name="value">
     <double>19.800000000000001</double>
    </property>
   </widget>
   <widget class="QLabel" name="lb32">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb33">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>z step</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="mSpnZPerStep">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>169</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="minimum">
     <double>-50.000000000000000</double>
    </property>
    <property name="maximum">
     <double>50.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.100000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lb34">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>169</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb35">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>z resolution factor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mEditZrf">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>20</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="mBtnUnpack">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>630</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Unpack</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mBtnReconstruct">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Reconstruct 2D series&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Reconstruct</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mBtnClose">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>630</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="mEditLog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>381</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QProgressBar" name="mProgress">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>680</y>
     <width>461</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="mBtnReconstruct3D">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>670</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Reconstruct with focal-line 3D algorithm&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Reconstruct 3D</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="mStatusBar">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Ready</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mBtnClearLog">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>420</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Log</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="ImageSliceDisplay" name="mImageDisplay" native="true">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>461</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="mBtnReconstructAverage">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>670</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Reconstruct single slice based on averaged channel data&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Reconstruct
Average</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mBtnSaveImage">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>630</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Image</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageSliceDisplay</class>
   <extends>QWidget</extends>
   <header>ImageSliceDisplay</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
